static/app/views/starfish/components/chart.tsx

```diff
@@ -284,6 +286,14 @@ function Chart({
               tooltip={areaChartProps.tooltip}
               colors={colors}
               grid={grid}
+              {...(showLineGraphChart
+                ? {
+                    legend: {
+                      top: 0,
+                      right: 10,
+                    },
+                  }
+                : {})}
```

Comment:
nit: This is cool lol but a bit unnecessary, I think we can just do something like:
And did you mean to call your variable something like `showLegend`?
```suggestion
              legend={showLineGraphChart ? {top: 0, right: 10} : undefined}
```

Reply:
Yeah good point @0Calories, I swear I tried that and it didn't work, but now it does lol.

Also, changed the variable name to `showLineGraphLegend`.